pkgs/os-specific/linux/odp-dpdk/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchgit, autoreconfHook, autoconf, automake, libtool
+, openssl, libpcap, dpdk
+}:
+
+stdenv.mkDerivation rec {
+  name = "odp-dpdk-${version}";
+  version = "1.8.0.0";
+
+  src = fetchgit {
+    url = "https://git.linaro.org/lng/odp-dpdk.git";
+    rev = "438a207a39bad213cdc03929452a8199caef5d8c";
+    sha256 = "0k4g5zbirbfdcgqz0nbn9san66y178qnigyvrr2apj3apzjjy7zv";
+  };
+
+  nativeBuildInputs = [ autoreconfHook ];
+  buildInputs = [ stdenv automake libtool openssl dpdk libpcap ];
+
+  RTE_SDK = "${dpdk}";
+  RTE_TARGET = "x86_64-native-linuxapp-gcc";
+
+  NIX_CFLAGS_COMPILE = [ "-march=core2" ];
+
+  patchPhase = ''
+    substituteInPlace scripts/git_hash.sh --replace /bin/bash /bin/sh
```

Comment:
Are you sure the script does not use bash specific stuff? If it does you should replace with bash from nix store